src/main/java/uk/gov/hmcts/ccd/domain/model/casedeletion/TTL.java

```diff
@@ -20,8 +25,14 @@
     public static final String YES = "Yes";
     public static final String NO = "No";
 
+    @JsonDeserialize(using = LocalDateDeserializer.class)
```

Comment:
I don't think you need these annotations, if you use the correctly configured ObjectMapper.

See

`ok, we've this setting to handle that https://github.com/hmcts/ccd-data-store-api/blob/master/src/main/java/uk/gov/hmcts/ccd/config/JacksonObjectMapperConfig.java#L29`

on following PR

`https://github.com/hmcts/ccd-data-store-api/pull/1679`

Reply:
Fixed in [7473a67](https://github.com/hmcts/ccd-data-store-api/pull/1712/commits/7473a672e3d99d8d3efc91e65db194e1785a959f)